### PR TITLE
Add capture

### PR DESCRIPTION
**Pull request by jacereda**

### CODE REVIEW

**Comment by jacereda** [comment]
I'm trying to implement a file selector (image selector really). I'm not versed in web development and I'm not sure `files` should go there, I added it to make the compiler happy.

**Comment by jacereda** [comment]
This PR is related to https://github.com/slamdata/purescript-halogen/pull/419

**Comment by jacereda** [comment]
And this is the code I'm trying to get running with these changes:

https://github.com/jacereda/purescript-halogen-fileselector/blob/master/src/Halogen/FileSelector/Component.purs

Any hint on how to remove the `unsafeCoerce` usages would be appreciated.

File: src/DOM/HTML/Indexed.purs [hunk 1]
```diff
@@ -328,6 +329,7 @@ type HTMLinput = Interactive
   ( accept :: MediaType
   , autocomplete :: OnOff
   , autofocus :: Boolean
+  , capture :: Boolean
```
Comment (natefaubion) [inline src/DOM/HTML/Indexed.purs]: Per https://w3c.github.io/html-media-capture/#the-capture-attribute, `capture` is a sensible enum. There should be an appropriate data type for this, like the other enumerated values.

```purescript
data CaptureValue
  = CaptureUser
  | CaptureEnvironment
```

Comment (jacereda) [inline src/DOM/HTML/Indexed.purs]: AFAICS that's a draft and the published spec says it's a boolean:

https://www.w3.org/TR/html-media-capture/#the-capture-attribute

**Comment by jacereda** [comment]
In any case, according to http://stackoverflow.com/questions/21523544/html-file-input-control-with-capture-and-accept-attributes-works-wrong#23360323 the `capture` attribute is pretty much useless at this point.

**Comment by natefaubion** [comment]
> In any case, according to http://stackoverflow.com/questions/21523544/html-file-input-control-with-capture-and-accept-attributes-works-wrong#23360323 the capture attribute is pretty much useless at this point.

Should this be closed then?

**Comment by jacereda** [comment]
Sure. Thanks.

**Pull request closed without merging**